src/components/dialogs/contingency-list/edition/contingency-list-edition-dialog.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSelector } from 'react-redux';
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import {
+    editContingencyList,
+    getEmptyFormData,
+    getFormDataFromFetchedElement,
+    getSchema,
+} from '../contingency-list-utils';
+import { useEffect, useMemo } from 'react';
+import { fetchContingencyList } from '../../../../utils/rest-api';
+import CustomMuiDialog from '../../custom-mui-dialog';
+import ContingencyListEditionForm from './contingency-list-edition-form';
+import { NAME } from '../../../utils/field-constants';
+
+const emptyFormData = getEmptyFormData();
```

Comment:
avoid to export "getEmptyFormData", "getSchema"  very generic name
otherwise, in the future, if someone types "getEmptyFormData", the IDE helper will give you many confusing suggestions

Reply:
done